docs/handbook/03_frames.md

```diff
@@ -5,7 +5,9 @@ description: "Turbo Frames decompose pages into independent contexts, which can
 
 # Decompose with Turbo Frames
 
-Turbo Frames are created by wrapping a segment of the page in a `<turbo-frame>` element. Each element must have a unique ID, which is used to match the content being replaced when requesting new pages from the server. A single page can have multiple frames, each establishing their own context:
```

Comment:
Note that this original paragraph is not removed, I just deleted the word "Turbo" to prevent repetition from the newly inserted paragraph.